azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participant/grid/ParticipantGridViewModel.kt

```diff
@@ -26,15 +25,13 @@ internal class ParticipantGridViewModel(
     private var updateVideoStreamsCallback: ((List<Pair<String, String>>) -> Unit)? = null
     private var remoteParticipantStateModifiedTimeStamp: Number = 0
     private var dominantSpeakersStateModifiedTimestamp: Number = 0
-    private var visibilityStatus: VisibilityStatus = VisibilityStatus.VISIBLE
+    private var visibilityStatus: VisibilityStatus? = null
```

Comment:
Why make nullable? Detracts from code safety. Is a default not sufficient?

Reply:
That was exactly a problem. Default value then compared to VisibilityStatus.VISIBLE does not give correct result when exiting from Pip